flopy/mf6/mfbase.py

```diff
@@ -454,24 +453,13 @@ class PackageContainer:
     modflow_models = []
     models_by_type = {}
 
-    def __init__(self, simulation_data, name):
-        self.type = "PackageContainer"
-        self.simulation_data = simulation_data
-        self.name = name
-        self._packagelist = []
+    def __init__(self, simulation_data):
+        self._simulation_data = simulation_data
```

Comment:
what's the reasoning for changing `simulation_data` -> `_simulation_data` here, and the opposite in `MFPackage`?